test/run

```diff
@@ -160,7 +180,7 @@ def print_counter(counter):
 
 def main():
     args = get_args()
-    logging.basicConfig(format="run-tests: %(message)s")
+    logging.basicConfig(format="\nrun-tests: %(message)s")
```

Comment:
Can I ask why you're adding a newline here?

Reply:
In case of this error or basically any logging statements in code - 

With `\n`
```
17 / 93 tests added
run-tests: Variable SLOW is missing from: custom_stages.py
```

Without `\n`
```
17 / 93 tests addedrun-tests: Variable SLOW is missing from: custom_stages.py
```